datastore/sync_entity.go

```diff
@@ -263,6 +293,164 @@ func (dynamo *Dynamo) InsertSyncEntitiesWithServerTags(entities []*SyncEntity) e
 	return nil
 }
 
+// DisableSyncChain marks a chain as disabled so no further updates or commits can happen
+func (dynamo *Dynamo) DisableSyncChain(clientID string) error {
```

Comment:
Shouldn't we cover this in a unit test in sync_entity_test.go?
Same for those new public methods in this file.